elide-integration-tests/src/test/java/com/yahoo/elide/errorEncodingTests/VerboseEncodedErrorResponsesIT.java

```diff
@@ -148,19 +146,4 @@ public void graphQLMutationError() {
             .statusCode(HttpStatus.SC_OK)
             .body(equalTo(expected));
     }
-
-    @Test
-    @Disabled
-    public void graphQLFetchError() {
```

Comment:
It is now doing the same encoding as `EncodedErrorResponseIT` because the error is now thrown by `GraphQLEntityProjectionMaker`, the verbose would not make difference.